### PR TITLE
Fix OIDC_ENABLE Toggle Logic in MISP Environment Configuration

### LINKED ISSUE #158: MISP Env `OIDC_ENABLE=true`
MISP Env `OIDC_ENABLE=true` not working correctly, where when doing `OIDC_ENABLE=true` and then `OIDC_ENABLE=false` it still remains with `OIDC_ENABLE=true`

### PR DESCRIPTION
**Pull request by Diogo-Rego**

### CODE REVIEW

File: core/files/configure_misp.sh [hunk 1]
```diff
@@ -72,46 +72,72 @@ GPGEOF
...
+                \"roles_property\": \"\",
+                \"role_mapper\": \"\",
+                \"default_org\": \"\"
+            }
+        }" > /dev/null
 
-    # Disable password confirmation as stated at https://github.com/MISP/MISP/issues/8116
-    sudo -u www-data /var/www/MISP/app/Console/cake Admin setSetting -q "Security.require_password_confirmation" false
+        # Use sed to remove the OidcAuth.Oidc entry from the 'auth' array in the config.php
+        sudo sed -i "/'auth' =>/,/)/ { /0 => 'OidcAuth.Oidc',/d; }" /var/www/MISP/app/Config/config.php
```
Comment (ostefano) [inline core/files/configure_misp.sh]: Use `sudo -u www-data` here

**Review by ostefano — CHANGES_REQUESTED** [review]
What if I have

File: core/files/configure_misp.sh (hunk 1 above)
Comment (Diogo-Rego) [reply core/files/configure_misp.sh]: @ostefano  like this?
```
# Use sed to remove the OidcAuth.Oidc entry from the 'auth' array in the config.php
sudo -u www-data sed -i "/'auth' =>/,/)/ { /0 => 'OidcAuth.Oidc',/d; }" /var/www/MISP/app/Config/config.php

# Remove the custom logout URL
sudo -u www-data sed -i "/'CustomAuth_custom_logout' =>/d" /var/www/MISP/app/Config/config.php
```

Comment (ostefano) [reply core/files/configure_misp.sh]: Yes, all commands that edit MISP state and file should use the same user/permission used by MISP

**Review by ostefano — APPROVED** [review]

**Pull request merged**